app/controllers/applications_controller.rb

```diff
@@ -9,7 +9,12 @@ def new
 
   def create
     application = Application.create(application_params)
-    redirect_to "/applications/#{application.id}"
+    if application.save
```

Comment:
This is a really interesting way to handle this! Very nice.

I don't _think_ it's possible because of our validations, but I wonder if it'd be safer to do Application.new and then check to save it? That way we don't create the record and then try to save it a second time? Not sure that's even a thing.

An alternative may be .new_record?